spec/property_spec.rb

```diff
@@ -110,6 +110,7 @@
     end
 
     @iface["ReadOrWriteMe"] = "VALUE"
+    @iface.SetTwoProperties("REAMDE", 255)
```

Comment:
np: REAMDE or README?

Reply:
It's a cultural joke :) https://en.wikipedia.org/wiki/Reamde